### PR TITLE
Sync all ticket notes when ticket is created by callback

### PR DESCRIPTION
**Pull request by kti-cameron**

### CODE REVIEW

File: djconnectwise/sync.py [hunk 1]
```diff
@@ -2215,9 +2225,11 @@ def sync_related(self, instance, sync_config={}):
         self.sync_children(*sync_classes)
 
     def fetch_sync_by_id(self, instance_id, sync_config={}):
-        instance = super().fetch_sync_by_id(instance_id)
+        api_instance = self.get_single(instance_id)
+        instance, result = self.update_or_create_instance(api_instance)
+
         if not instance.closed_flag:
-            self.sync_related(instance, sync_config)
+            self.sync_related(instance, sync_config, result)
```
Comment (kti-daniel) [inline djconnectwise/sync.py:2232]: Is the varialbe, `result` is used elsewhere? It seems passed but not returned or used. Or Is the source up to date?

**Comment by kti-daniel** [comment]
We have 2 kinds of notes such as general notes and time entry notes, so I feel we could apply the same thing to time entry notes? What about your idea?

image

File: djconnectwise/sync.py (hunk 1 above)
Comment (kti-daniel) [reply djconnectwise/sync.py:2232]: Oh, my confusion. Never mind~ Got it.

**Comment by kti-cameron** [comment]
Good catch! Thanks for thinking of it.

**Comment by kti-daniel** [comment]
Looks good~ I will do a final test and merge this in.

**Pull request merged**